frontend/src/posapp/components/navbar/AboutDialog.vue

```
@@ -91,70 +91,74 @@
+	emit("update:modelValue", val);
+});
+
+function close() {
+	dialogOpen.value = false;
+}
+
+function loadAppInfo() {
+	loadingAppInfo.value = true;
+	appInfoError.value = false;
+
+	frappe.call({
```

ESLint

[error] 'frappe' is not defined.

(no-undef)